doc/docs/Python_User_Interface.md

```diff
@@ -534,29 +530,14 @@ def mean_time_spent_on(self, time_sink):
 <div class="method_docstring" markdown="1">
 
 Return the mean time spent by all processes for a type of work `time_sink` which
-can be one of the following integer constants:
-* meep.Stepping ("time stepping")
-* meep.Connecting ("connecting chunks")
-* meep.Boundaries ("copying boundaries")
-* meep.MpiAllTime ("all-all communication")
-* meep.MpiOneTime ("1-1 communication")
-* meep.FieldOutput ("outputting fields")
-* meep.FourierTransforming ("Fourier transforming")
-* meep.MPBTime ("MPB mode solver")
-* meep.GetFarfieldsTime ("far-field transform")
-* meep.FieldUpdateB ("updating B field")
-* meep.FieldUpdateH ("updating H field")
-* meep.FieldUpdateD ("updating D field")
-* meep.FieldUpdateE ("updating E field")
-* meep.BoundarySteppingB ("boundary stepping B")
-* meep.BoundarySteppingWH ("boundary stepping WH")
-* meep.BoundarySteppingPH ("boundary stepping PH")
-* meep.BoundarySteppingH ("boundary stepping H")
-* meep.BoundarySteppingD ("boundary stepping D")
-* meep.BoundarySteppingWE ("boundary stepping WE")
-* meep.BoundarySteppingPE ("boundary stepping PE")
-* meep.BoundarySteppingE ("boundary stepping E")
-* meep.Other ("everything else")
+can be one of the following integer constants: `0`: "time stepping", `1`: "connecting chunks",
```

Comment:
Why do we use these integer values here instead of the enum?

Reply:
Two reasons: (1) integer values for the indices of the timing stack was the way this was originally documented and (2) for convenience, since the enums sometimes have too many characters to type out.